sympy/functions/combinatorial/factorials.py

```diff
@@ -210,9 +210,11 @@ class subfactorial(CombinatorialFunction):
         !n = {  0                             for n = 1
              |  (n - 1)*(!(n - 1) + !(n - 2)) for n > 1
               `
+    recurrence can be reduced to 
 
-    It can also be written as int(round(n!/exp(1))) but the recursive
-    definition with caching is implemented for this function.
+    !n = n*( !(n-1) ) + (-1)**n
```

Comment:
You should also add a note, regarding which formula is used for computing subfactorial.